src/test/regress/input/multi_copy.source

```diff
@@ -497,7 +497,10 @@ SELECT shardid, nodename, nodeport
 	WHERE logicalrelid = 'numbers_append'::regclass order by placementid;
 
 -- add the node back
-SET client_min_messages TO ERROR;
+-- before adding the node, add the pg_dist_object entry for objects table
+-- manually. Since that table was created with master_create_distributed_table
+-- we don't have the entry for it.
```

Comment:
I've added that to manually create entry on citus.pg_dist_object for table created with master_create_distributed_table. I don't update the CreateDistributedTable to add objects for such tables because of 3 reasons.
1- We've deprecated that API
2- Even if we start to add such tables as objects it doesn't resolve the problem of upgrade. I think we should create such manual entries for tables created with master_create_distributed_table within upgrade scripts.
3- It breaks loots of tests :)